test/helpers/mocks.cr

```diff
@@ -106,6 +114,38 @@ class EchoJob < Mosquito::QueuedJob
   end
 end
 
+class RateLimitedJob < Mosquito::QueuedJob
+  include Mosquito::RateLimiter
+
+  throttle key: "rate_limit", limit: Int32::MAX
+
+  params should_fail : Bool = false, increment : Int32 = 1
+
+  before do
+    log "Before Hook Executed"
+    fail if should_fail
+  end
+
+  def perform
+    log "Performed"
+  end
+
+  def increment_run_count_by
+    increment
+  end
+end
+
+class SecondRateLimitedJob < Mosquito::QueuedJob
+  include Mosquito::RateLimiter
+
+  throttle key: "rate_limit"
+
+  params()
```

Comment:
```suggestion
  # TODO: https://github.com/mosquito-cr/mosquito/issues/66
  params()
```

Just a suggestion, but it'll at least explain why and not just look like random floating code.

Reply:
You know I think of issue #66 a lot these days... 🤦‍♂️